meta-mender-core/classes/mender-artifactimg.bbclass

```diff
@@ -18,6 +18,31 @@ ARTIFACTIMG_FSTYPE_DEFAULT = "ext4"
 ARTIFACTIMG_NAME ??= "${ARTIFACTIMG_NAME_DEFAULT}"
 ARTIFACTIMG_NAME_DEFAULT = "${IMAGE_LINK_NAME}"
 
+MENDER_ARTIFACT_NAME_DEPENDS ?= ""
+
+MENDER_ARTIFACT_PROVIDES ?= ""
+MENDER_ARTIFACT_PROVIDES_GROUP ?= ""
+
+MENDER_ARTIFACT_DEPENDS ?= ""
+MENDER_ARTIFACT_DEPENDS_GROUPS ?= ""
+
+apply_arguments () {
+    #
+    # $1 -- the command line flag to apply to each element
+    # $@ -- the list of arguments to give each its own flag
+    #
+    local res=""
+    flag=$1
+    shift
+    OIFS=$IFS
+    IFS=" "
```

Comment:
Is this necessary? I think `IFS` already breaks on spaces if you don't specify it. Also, despite looking correct, I don't think that

```
OIFS="$IFS"
IFS="$OIFS"
```

resets it to the same state, because empty `IFS` is not the same as unset `IFS`, and in that sequence of commands the shell can't tell the difference. Not 100% sure though.